src/content/docs/en/recipes/dynamically-importing-images.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Dynamically Import Images
+description: Learn how to dynamically import images using Vite's import.meta.glob function
+i18nReady: true
+type: recipe
+---
+import FileTree from '~/components/FileTree.astro'
+
+When working with [images](/en/guides/images/), there will be times where you will need to dynamically import the image paths. 
+
+In this recipe, you will learn how to dynamically import your images the correct way using Vite's `import.meta.glob` function. 
+
+## Recipe
+
+1. Create a new `assets` folder under the `src` directory and add your images inside that new folder. 
+
+<FileTree>
+- src/
+  - assets/
+    - example-image.jpg
+    - example-image-2.jpg
+    - example-image-3.jpg
+</FileTree>
+
+:::note
+`assets` is a popular folder name convention for placing images but you are free to name the folder whatever you like.
+::: 
+
+2. Create a new Astro component and import the `<Image>` component.
+
+    ```astro title="src/components/MyCustomCardComponent.astro" 
+    ---
+     import { Image } from 'astro:assets';
+    ---
+    ```
+
+3. Create the `props` for your component. 
```

Comment:
```suggestion
3. Specify the `props` that your component will receive in order to display the necessary information on each card. You can optionally define their types, if you are using TypeScript in your project.
```
I'm thinking something like this is again helpful to reinforce the narrative. (It can be hard to just say, "Here are your props!" but since we've already established what we're trying to build, this is just a gentle reminder, "Hey, we said we were going to display name, age and an image! So, we're setting up the component to be ready to handle that info!"

I love typing the props here! And I think I'd like to call attention to it as optional since I know many of our users aren't using TypeScript. See what you think of the above.